### ISSUE TITLE
Crash w/ Lord of the Rings Mod

### ISSUE THREAD

**Issue by WenXin20** [PianoManu/BlockCarpentry#2]
`Minecraft v1.15.2`
`Forge v31.2.36`
`Block Carpentry v0.6.0`
[`Lord of the Rings v1.7.1`](https://lotrminecraftmod.fandom.com/wiki/Updates/Renewed)(Download is under snapshot 1.7)

I had placed the carpentry vertical slab, but when I right-clicked it with the Drystone from the lotr mod, the game crashed immediately. 
[Crash Report](https://gist.github.com/WenXin20/91b581f00b2a53af54b41e22143b54b2)

**Comment by PianoManu** [comment]
Yup, same error as #1 , because these are non-vanilla blocks and the blocks are currently hard-coded. I'm working on a solution ^^
Thanks for reporting

**Comment by WenXin20** [comment]
Its not quite the same crash, the other one was a server only crash, but this one crashed me client-side only to the desktop.

**Comment by PianoManu** [comment]
Indeed, it's a different one. And it looks strange. I will fix it ASAP, but it might take some time since it looks pretty difficult...

**Comment by WenXin20** [comment]
It's most likely because that mod has its own connected textures system and doesn't know what to do when placed in the carpentry blocks.
For now, I'll just avoid using blocks from that mod with it.

**Comment by WenXin20** [comment]
I just tested this again, the crash still happens with [Lotr v2.2](https://www.curseforge.com/minecraft/mc-mods/the-lord-of-the-rings-mod-renewed) and this mod.
[Crash Report](https://gist.github.com/WenXin20/d399067ece07f5b03ba8b906d910df11)

**Comment by PianoManu** [comment]
Is this the whole report? Because for me it ends with Line 6763
`[17:49:39] [Render thread/WARN]: [OptiFine] Missing CTM sprite: minecraft:missingno, properties: optifine/ctm/minecraft/podzol_side`
and there is no crash or any sort of fatal error 🤷‍♂️ 

To be honest, I really don't know how to fix the original bug (connected texture causes crash), that's why I added the **help wanted** label

**Comment by WenXin20** [comment]
Whoops, I accidentally sent the latest.log instead. The link should be fixed now.
